.github/ISSUE_TEMPLATE/documentation.yaml

```
@@ -1,15 +1,34 @@
+  attributes:
+    label: Type of documentation issue
+    options:
+      - Missing information
+      - Unclear explanation
+      - Outdated content
+      - Formatting/structure improvement
+      - Other (please specify in the description)
+  validations:
+    required: true
+  - type: input
+  id: location
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)